src/modules/keepalive/keepalive_mod.c

```diff
@@ -126,6 +138,8 @@ static int mod_init(void)
  */
 static void mod_destroy(void)
 {
+	lock_release(ka_destinations_list->lock);
+	lock_dealloc(ka_destinations_list->lock);
 }
```

Comment:
This two have to be enclosed in `if(ka_destinations_list) { ... }`, the mod_destroy() is executed even when mod_init() is not finished, in that case ka_destinations_list is NULL, crashing kamailio at shutdown.